<robot name="obstacle">
        <link name="my_obstacle">
            <inertial>
                <origin xyz="0 0 4.5" />
                <mass value="10.0" />
                <inertia  ixx="0.0" ixy="0.0"  ixz="0.0"  iyy="0.0"  iyz="0.0"  izz="0.0" />
            </inertial>
            <visual>
                <origin xyz="0 0 4.5"/>
                <geometry>
                <cylinder radius="1" length="9"/>
                </geometry>
            </visual>
            <collision>
                <origin xyz="0 0 4.5"/>
                <geometry>
                <box size="0.1 0.1 0.1" />
                </geometry>
            </collision>
        </link>
        <gazebo reference="my_obstacle">
            <material>Gazebo/Blue</material>
        </gazebo>
    </robot>